pkgs/top-level/aliases.nix

```diff
@@ -590,6 +591,7 @@ mapAliases ({
   k3d = throw "k3d has been removed because it was broken and has seen no release since 2016"; # Added 2022-01-04
   k9copy = throw "k9copy has been removed from nixpkgs, as there is no upstream activity"; # Added 2020-11-06
   kafkacat = kcat; # Added 2021-10-07
+  kalendar = libsForQt5.kdeGear.kalendar; # Added 2022-04-21
```

Comment:
There is a section below for plasma5Packages. I think it should be placed there instead. 

https://github.com/NixOS/nixpkgs/blob/0b68633bde03e260fb08a118cad295ea33118dfb/pkgs/top-level/aliases.nix#L1496-L1499

Reply:
Done.

Reply:
Also for `skanpage`.